modules/sr/robot/robot.py

```diff
@@ -9,8 +9,13 @@
 from controller import Robot as WebotsRobot  # isort:skip
 
 
-class Robot:
-    """Class for initialising and accessing robot hardware"""
+class ManualTimestepRobot:
+    """
+    Class for initialising and accessing robot hardware.
```

Comment:
Minor point: “Class for” contains no information; we can see this is a class by the class keyword above.

Reply:
So I was wondering what the `sr-robot` did any better and it turns out that this is from there.

Reply:
1050942